aspnetcore/fundamentals/index.md

```diff
@@ -16,97 +16,121 @@ ms.custom: H1Hack27Feb2017
+In ASP.NET Core, you compose your request pipeline using [Middleware](xref:fundamentals/middleware). ASP.NET Core middleware performs asynchronous logic on an `HttpContext` and then either invokes the next middleware in the sequence or terminates the request directly. You generally "Use" middleware by taking a dependency on a NuGet package and invoking a corresponding `UseXYZ` extension method on the `IApplicationBuilder` in the `Configure` method.
 
 ASP.NET Core comes with a rich set of built-in middleware:
 
-* [Static files](static-files.md)
+* [Static files](xref:fundamentals/static-files)
 
-* [Routing](routing.md)
+* [Routing](xref:fundamentals/routing)
 
-* [Authentication](../security/authentication/index.md)
+* [Authentication](xref:security/authentication/index)
 
 You can use any [OWIN](http://owin.org)-based middleware with ASP.NET Core, and you can write your own custom middleware.
 
-For more information, see [Middleware](middleware.md) and [Open Web Interface for .NET (OWIN)](owin.md).
+For more information, see [Middleware](xref:fundamentals/middleware) and [Open Web Interface for .NET (OWIN)](xref:fundamentals/owin).
 
 ## Servers
 
-The ASP.NET Core hosting model does not directly listen for requests; rather it relies on an HTTP server implementation to forward the request to the application. The forwarded request is wrapped as a set of feature interfaces that the application then composes into an `HttpContext`.  ASP.NET Core includes a managed cross-platform web server, called [Kestrel](servers/kestrel.md) that you would typically run behind a production web server like [IIS](https://iis.net) or [nginx](http://nginx.org).
+The ASP.NET Core hosting model does not directly listen for requests; rather, it relies on an HTTP server implementation to forward the request to the application. The forwarded request is wrapped as a set of feature interfaces that the application then composes into an `HttpContext`. ASP.NET Core includes a managed, cross-platform web server, called [Kestrel](xref:fundamentals/servers/kestrel), that you would typically run behind a production web server like [IIS](https://iis.net) or [nginx](http://nginx.org).
 
-For more information, see [Servers](servers/index.md) and [Hosting](hosting.md).
+For more information, see [Servers](xref:fundamentals/servers/index) and [Hosting](xref:fundamentals/hosting).
 
 ## Content root
 
-The content root is the base path to any content used by the app, such as its views and web content. By default the content root is the same as application base path for the executable hosting the app; an alternative location can be specified with *WebHostBuilder*.
+The content root is the base path to any content used by the application, such as its views and web content. By default, the content root is the same as application base path for the executable hosting the application; an alternative location can be specified with `WebHostBuilder`.
```

Comment:
The content root is the base path to any content used by the app, such as views, Razor Pages and static assets. By default, the content root is the same as application base path for the executable hosting the application. An alternative location for content root is specified with `WebHostBuilder`.

avoid future tense